src/JuliusSweetland.OptiKey.Core/Observables/TriggerSources/PointFixationSource.cs

```diff
@@ -133,6 +133,7 @@ public IObservable<TriggerSignal> Sequence
                                         //Publish a high signal if progress is 1 (100%), otherwise just publish progress (filter out 0 as this is a progress reset signal)
                                         if (progress > 0)
                                         {
+                                            fixationCentrePointAndKeyValue = point.Value;
```

Comment:
Things may still be in progress, but can you remove Kirsty's commit from this PR? Thanks.

Reply:
Done and my comment edited to new commit.